presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java

```diff
@@ -27,6 +27,17 @@
     private final Duration elapsedTime;
     private final Duration queuedTime;
 
+    /**
+     * Takes the waiting time and the elapsed time.
```

Comment:
This is not the purpose of the method. What it really does with these times?
What does this DispatchInfo means?